### ISSUE TITLE
About predict_peaks()

### ISSUE THREAD

**Issue by ccllabe** [bihealth/NeatMS#9]
Hello, I have a queation about running NeatMS code. When I run to predict_peaks(), show the error below:
ValueError: Unexpected result of `predict_function` (Empty batch_outputs). Please use `Model.compile(..., run_eagerly=True)`, or `tf.config.run_functions_eagerly(True)` for more information of where went wrong, or file a issue/bug to `tf.keras`.
How can I fix this problem, thanks.

**Comment by yoglo** [comment]
Hi, are you using a default model, or did you train your own? 
It may be coming from a problem with the dimensions of the data that is passed as input not matching the dimensions the model expects - this might happen when the model is retrained from scratch with an architecture different from the base model.

**Comment by ccllabe** [comment]
> Hi, are you using a default model, or did you train your own? It may be coming from a problem with the dimensions of the data that is passed as input not matching the dimensions the model expects - this might happen when the model is retrained from scratch with an architecture different from the base model.

Hello, I wonder how to get the raw data, I only have wiff format file. Is there a way to convert wiff file to raw file, thanks.

**Comment by yoglo** [comment]
Yes, you can convert your files to mzML format using MSconvert from ProteoWizard ([this should help](https://ccms-ucsd.github.io/GNPSDocumentation/fileconversion/)). You can also check out data format requirements on NeatMS doc [here](https://neatms.readthedocs.io/en/latest/first-steps/data-format/)
